src/Libraries/Microsoft.Extensions.AsyncState/AsyncState.cs

```diff
@@ -104,7 +104,9 @@ internal static void EnsureCount(List<object?> features, int count)
         }
     }
 
+#pragma warning disable CA1822
```

Comment:
It would be good to add a comment explaining what this warning do, e.g.
```suggestion
#pragma warning disable CA1822 // Comments goes here....
```

Reply:
I added explanation comment